test/core/simd/simd_i16x8.wast

```diff
@@ -0,0 +1,387 @@
+;; Tests for i16x8 [min_s, min_u, max_s, max_u] operations.
```

Comment:
Should the file name include "min" and "max" or is it going to eventually include many more operations?

Reply:
Not any good ideas. The best I can think of is `simd_i16x8_val_arith.wast`. Adding "val" doesn't really convey much more information, but it does suggest that the contents are more specific than the name `simd_i16x8.wast` suggests. My other idea is `simd_i16x8_arith2.wast`, which has the same benefits and drawbacks.

Reply:
I'd prefer `simd_i16x8_arith2.wast`. :)